lib/runner/concurrency/child-process.js

```diff
@@ -51,7 +51,7 @@ class ChildProcess extends EventEmitter {
    * @returns {Array}
    */
   getArgs() {
-    let args = [this.mainModule];
+    let args = [...process.execArgv, this.mainModule];
```

Comment:
The only thing we need to test is this change, which could be done by calling only this method instead of running all the unit tests with --inspect, which is quite a big change.